pipeline-scripts/create-database.sql

```diff
@@ -952,4 +953,25 @@ DO
 DELETE FROM auth_success WHERE Expiry < UTC_TIMESTAMP();
 //
 
+-- --------
+-- --------
+-- Stored Procedures
+-- --------
+-- --------
+
+CREATE PROCEDURE UpdateMarketSummaryLiquidityPoolCount(IN marketId BIGINT UNSIGNED, IN blockHeight BIGINT UNSIGNED)
```

Comment:
Need to be able to differentiate between updating all markets (for example during rewinds) and a single market (when a pool is created). There's no concept of optional/default params in MySQL so we have to provide 0 as a market id to the proc if we want to run the update against all pools.